security.php

```diff
@@ -16,6 +16,13 @@
 require_once __DIR__ . '/security/login-error.php';
 require_once __DIR__ . '/security/password.php';
 
+if ( defined( 'VIP_SECURITY_INACTIVE_USERS_ACTION' ) ) {
+	require_once __DIR__ . '/security/user-last-seen.php';
+
+	$last_seen = new \Automattic\VIP\Security\User_Last_Seen();
+	$last_seen->init();
+}
```

Comment:
I think maybe we should load all this on the `plugins_loaded` hook, to allow for sites to have more time to conditionally setup the constants and use the filters.

That said, need to make sure it's still early enough to catch any of the authenticate calls.